### PR TITLE
2/2: libidn rebuild (VLC, pidgin, libgsasl, Lynx)

### PR DESCRIPTION
**Pull request by Mno-hime**
Merge after https://github.com/OpenIndiana/oi-userland/pull/4827.

Updated Lynx to latest release, made it 64-bit. Test suite passed. https://www.howsmyssl.com/ report "Probably Good", IDN works.

### CODE REVIEW

**Comment by pyhalov** [comment]
libgsasl also needs rebuilding.
system/management/web/openwsman should be either rebuilt or dropped (likely, dropped)

**Comment by Mno-hime** [comment]
> system/management/web/openwsman should be either rebuilt or dropped (likely, dropped)

I don't see this one on oi-userland. ?

**Comment by pyhalov** [comment]
It is legacy package, inherited from OI /dev, it has been never added to oi-userland.

File: components/meta-packages/history/history [hunk 1]
```diff
@@ -1109,6 +1109,7 @@ system/management/visual-panels/panel-time@0.5.11,5.11-2015.0.2.0
 system/management/visual-panels/panel-usermgr@0.5.11,5.11-2015.0.2.0
 system/management/visual-panels@0.5.11,5.11-2015.0.2.0
 system/management/wbem/resource-management@0.5.11,5.11-2015.0.2.0
+system/management/web/openwsman@2.1.0-2013.0.0.0
```
Comment (pyhalov) [inline components/meta-packages/history/history]: Mark it with  2018.0.0.0, as 2013 is reserved for 'not-rebuilt legacy'

**Pull request merged**